src/Command/GenerateObjectAclCommand.php

```diff
@@ -62,7 +62,7 @@ public function __construct(Pool $pool, array $aclObjectManipulators, $registry
     {
         $this->pool = $pool;
         $this->aclObjectManipulators = $aclObjectManipulators;
-        if (null !== $registry && !(method_exists($registry, 'getAliasNamespace') || method_exists($registry, 'getEntityNamespace'))) {
+        if (null !== $registry && (!$registry instanceof RegistryInterface || !$registry instanceof ManagerRegistry)) {
```

Comment:
Does this work if one of the class does not exist in codebase? Because this is the case here. That was the reason I was checking for the message.

Reply:
[Yes](https://3v4l.org/VfiIR)

Reply:
Wow cool. good to know